diff --git a/node_modules/react-native-contacts/index.d.ts b/node_modules/react-native-contacts/index.d.ts
index b3a381a..ef1d68d 100644
--- a/node_modules/react-native-contacts/index.d.ts
+++ b/node_modules/react-native-contacts/index.d.ts
@@ -22,6 +22,7 @@ export interface EmailAddress {
 export interface PhoneNumber {
     label: string;
     number: string;
+    foundUser?: { id: string; name: string; onesignal_id: string };
 }
 
 export interface PostalAddress {
