diff --git a/node_modules/react-native-sound/index.d.ts b/node_modules/react-native-sound/index.d.ts
index f505c91..a035580 100644
--- a/node_modules/react-native-sound/index.d.ts
+++ b/node_modules/react-native-sound/index.d.ts
@@ -111,7 +111,7 @@ declare class Sound {
    * Set the volume
    * @param value - ranging from 0.0 (silence) through 1.0 (full volume)
    */
-  setVolume(value: number): void
+  setVolume(value: number): this
 
   /**
    * Return the stereo pan position of the audio player (not the system-wide pan)
